src/data/faq_de.json

```diff
@@ -376,6 +379,216 @@
+                                ]
+                            },
+                            {
+                                "title": "Einige Tage nach dem letzten positiven Test bin ich beim Selbsttest noch immer positiv. Soll ich nochmal warnen?",
+                                "anchor": "warn_without_tan_still_positive",
+                                "active": false,
+                                "textblock": [
+                                    "Wir empfehlen weiterhin, dass Sie sich nach einem positiven Selbsttest eigenverantwortlich so vernünftig verhalten, dass Sie andere nicht anstecken. Wenn Sie sich so verhalten, ist eine weitere Warnung nicht notwendig.",
+                                    "Um zu verhindern, dass die Funktion missbraucht wird und sinnlose Warnungen ausgelöst werden, die nicht auf realen Risikokontakten beruhen, wurde beschlossen, dass ein Nutzer mit der neuen Funktion andere nur einmal innerhalb eines bestimmten Zeitraums warnen kann.",
+                                    "Nach Ablauf dieser Zeitspanne ist es dann wieder möglich, aufgrund eines eigenen positive Testergebnisses ohne TAN zu warnen. Diese Einschränkung gilt nicht für positive Testergebnisse, die per App übermittelt werden."
+                                ]
+                            },
+                            {
+                                "title": "Muss ich ein Foto meines Tests machen, um den Test nachzuweisen?",
+                                "anchor": "warn_without_tan_photo",
+                                "active": false,
+                                "textblock": [
+                                    "Nein, in der Corona-Warn-App ist nicht vorgesehen, Fotos oder andere Dokumente zum Nachweis, dass ein Test wirklich stattgefunden hat, hochzuladen oder anderweitig zu prozessieren. Wir setzen hier bewusst auf die Umsicht und Eigenverantwortung unserer Anwender*innen."
+                                ]
+                            },
+                            {
+                                "title": "Wie oft kann ich ohne TAN warnen?",
+                                "anchor": "warn_without_tan_how_often",
+                                "active": false,
+                                "textblock": [
+                                    "Innerhalb eines bestimmten Zeitraums können Sie einmal ohne TAN warnen. Nach Ablauf der Sperrfrist können Sie erneut ohne TAN warnen. Grundsätzlich gibt es nach oben keine Grenzen, wie oft Sie ohne TAN warnen können, solange die Abstände nicht unterschritten werden."
+                                ]
+                            },
+                            {
+                                "title": "Kann ich in Vertretung von Familienmitgliedern warnen, z.B. für mein infiziertes Kind?",

```

Comment:
"z.B." => `z.&nbsp;B.`

Reply:
Fixed in https://github.com/corona-warn-app/cwa-website/pull/3322/commits/f2b4444eace8ae8143537742d5f8e4f971a82b60, thanks!